ui/lib/core/addon/components/replication-actions.js

```diff
@@ -10,7 +10,6 @@ const DEFAULTS = {
   primary_cluster_addr: null,
   errors: [],
   id: null,
-  replicationMode: null,
```

Comment:
This is the fix for the replication actions that were essentially stalling out on most all actions.

Reply:
yeeehawww, nice find~!